code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -142,3 +142,38 @@
 	icon_state = "pill18"
 	list_reagents = list("insulin" = 50)
 	roundstart = 1
+
+/obj/item/weapon/genderpill //its a "pill"
+	name = "strange red pill"
+	desc = "The mythical redpill on gender issues. Do not snort."
+	icon_state = "pill4"
+	var/used = FALSE
+	var/newgender = null
+	force = 0
+	
+/obj/item/weapon/genderpill/attack_self(mob/user)
+	return
+	
+/obj/item/weapon/genderpill/attack(mob/M, mob/user, def_zone)
+	if(used)
+		return
+	used = 1
+	if(M == user)
+		if(istype(user, /mob/living/carbon))	
+			var/mob/living/carbon/H = user
+			if(H.gender == FEMALE)
+				H.gender = MALE
+				newgender = pick("dude", "man", "boy", "bro")
+			else
+				H.gender = FEMALE
+				newgender = pick("dudette", "woman", "girl", "brodette")
+			H.dna.update_ui_block(DNA_GENDER_BLOCK)
+			H.update_body()
+			H.update_mutations_overlay()
+			H.visible_message("<span class='notice'>[user] swallows the [src] and morphs into a [newgender]!</span>")
+			qdel(src)
+		else
+			user.visible_message("<span class='notice'>[user] swallows the [src] but isnt complex enough to understand gender issues!</span>")
+	else
+		M.visible_message("<span class='notice'>[user] attempts to feed [M] the [src] but it turns blue!  Looks like [M] wasn't ready to see from the other side.</span>")
+		qdel(src)

```

Comment:
why does it delete itself if you try to feed it to someone else, just have it not work